src/index.js

```diff
@@ -115,6 +120,24 @@ class LocalstackPlugin {
     // Activate the synchronous parts of plugin config here in the constructor, but
     // run the async logic in enablePlugin(..) later via the hooks.
     this.activatePlugin(true);
+
+    // If we're using webpack, we need to make sure we retain the compiler output directory
+    if (this.detectTypescriptPluginType() === TS_PLUGIN_WEBPACK) {
+      const p = this.serverless.pluginManager.plugins.find((x) => x.constructor.name === TS_PLUGIN_WEBPACK);
+      if (
+        this.shouldMountCode() && (
+          !p ||
```

Comment:
nitpick: Not critical, but I guess we could create a small util function like:
`extractPath(p, 'serverless.configurationInput.custom.webpack.keepOutputDirectory')` which would split the path into the individual parts and return the result (or `undefined`, if the path does not exist).

(This is to remain backward compatibility, as [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) seems to be only available in Node.js 14+).

Reply:
Agreed. Though, updating to node14 should be considered now that lambda supports it, so it's reasonable to expect anyone writing new lambda code is going to adopt it.